website/content/tools/autoscaling/cli.mdx

```diff
@@ -172,6 +175,12 @@ passed in via CLI arguments. The `agent` command accepts the following arguments
   hold a lock over a Nomad variable and will only execute if the lock is
   successfully acquired. It defaults to `false`.
 
+- `-high-availability-lock-namepsace`: Defines the path of the Nomad namespace
+  where the variable that will be used to sync the leader when running in high
+  availability mode. This parameter needs to be the same on all Nomad
+  Autoscaler agents expected to take part in the same leadership election
+  process. Defaults to `default`.
```

Comment:
Sneaking this one here because I forgot to do it in https://github.com/hashicorp/nomad/pull/19769 😅